package.json

```diff
@@ -11,9 +11,9 @@
 		"email": "admin@tri.be"
 	},
 	"browserslist": [
-		"last 2 versions",
```

Comment:
This was [excluding all versions of Safari under v18.0](https://browsersl.ist/#q=last+2+versions%2C%0Anot+op_mini+all%2C%0Anot+dead), which is bad as not many people have migrated to the latest version of MacOS/iOS yet.